.drone.yml

```diff
@@ -1,9 +1,57 @@
-workspace:
-  base: /go
-  path: src/github.com/drone/drone-go
-
-pipeline:
-  build:
-    image: golang:1.6
-    commands:
-      - go test -v github.com/drone/drone-go/drone
+---
+kind: pipeline
+type: docker
+name: testing
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: staticcheck
+  pull: always
+  image: golang:1.15
+  commands:
+  - go run honnef.co/go/tools/cmd/staticcheck ./...
+  volumes:
+  - name: gopath
+    path: /go
+
+- name: lint
+  pull: always
+  image: golang:1.15
+  commands:
+  - go run golang.org/x/lint/golint -set_exit_status ./...
+  volumes:
+  - name: gopath
+    path: /go
+
+- name: vet
+  pull: always
+  image: golang:1.15
+  commands:
+  - go vet ./...
+  volumes:
+  - name: gopath
+    path: /go
+
+- name: test
+  pull: always
+  image: golang:1.15
+  commands:
+  - go test -cover -v ./...
+  volumes:
+  - name: gopath
+    path: /go
+
+volumes:
+- name: gopath
+  temp: {}
+
+trigger:
```

Comment:
AFAIK Brad doesn't like to have this trigger within the drone org, this appears only within the drone-plugins org.